### PR TITLE
Migrate showcase to new control flow

### PR DESCRIPTION
**Pull request by marcjulian**
Added prettier for consistent formatting with the new Angular Control Flow.

Schematics for new control flow: https://angular.dev/reference/migrations/control-flow#

`ng generate @angular/core:control-flow`

### CODE REVIEW

**Comment by marcjulian** [comment]
The only component in the library using `NgIf` and `NgFor` is `MarkersForClustersComponent`. 
Migrating the component is straight forward, but when viewed in the demos (http://localhost:4200/demo/ngx-cluster-html) an error is thrown in the console when scrolling the map with clusters:

CleanShot 2024-06-03 at 11 59 44

**Comment by marcjulian** [comment]
Strange the unit test are failing for me even in the main branch. Not sure whats going on.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```
Comment (HarelM) [inline .prettierrc:2]: Can we do it with lint instead?

Comment (marcjulian) [reply .prettierrc:2]: This is just the config to match the current formatting styles, but prettier also supports formatting Angular's control flow (https://prettier.io/blog/2023/11/13/3.1.0.html#angular)

**Comment by marcjulian** [comment]
> Might be a chrome version change that doesn't load webgl when in headless mode. This is just a thought according to the error message and what you said.

You might be right. Last successful unit test in ci (https://github.com/maplibre/ngx-maplibre-gl/actions/runs/8726346554/job/23941372299) used `Chrome Headless 123.0.6312.124 (Mac OS 10.15.7)`. 

The failing ci test uses `Chrome Headless 125.0.6422.113 (Mac OS 10.15.7)` and locally I have `Chrome Headless 125.0.6422.141 (Mac OS 10.15.7)` both failing the same tests.

File: .prettierrc (hunk 1 above)
Comment (HarelM) [reply .prettierrc:2]: I prefer not to add another dependency if it is not required...

**Comment by HarelM** [comment]
Let's remove prittier and merge this, we can discuss if we want to introduce prittier in another PR/issue/discussion.

File: .prettierrc (hunk 1 above)
Comment (marcjulian) [reply .prettierrc]: okay

**Review by HarelM — APPROVED** [review]

**Pull request merged**